kotlin-detect-config.yaml

```diff
@@ -202,23 +219,28 @@ exceptions:
             - "**/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/setuppayloadscanner/BarcodeFragment.kt"
             - "**/examples/java-matter-controller/java/src/com/matter/controller/commands/common/Argument.kt"
             - "**/examples/java-matter-controller/java/src/com/matter/controller/Main.kt"
+            - "**/src/controller/java/generated/java/**/*"
             - "**/src/controller/java/tests/chip/onboardingpayload/ManualCodeTest.kt"
             - "**/src/controller/java/tests/chip/onboardingpayload/QRCodeTest.kt"
     SwallowedException:
         excludes:
             - "**/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/SensorClientFragment.kt"
             - "**/examples/java-matter-controller/java/src/com/matter/controller/commands/common/Argument.kt"
+            - "**/src/controller/java/generated/java/**/*"
```

Comment:
Do we swallow exceptions in generated code? Why?

Reply:
I removed.